2017-05-11T14:11:34.174Z - error: Error during processing request: http://localhost:3000/reporting/api/report/Factura details: Could not parse report template, aren't you missing content type? undefined
2017-05-11T14:11:45.917Z - error: Error during processing request: http://localhost:3000/reporting/api/report/Factura details: Could not parse report template, aren't you missing content type? undefined
2017-05-11T14:13:03.828Z - error: Error during processing request: http://localhost:3000/reporting/api/report/Factura details: Could not parse report template, aren't you missing content type? undefined
2017-05-11T14:23:13.698Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:23:13 GMT-0300 (Hora estándar de Argentina), pid=7108, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134119424, heapTotal=110481408, heapUsed=72468624, external=19365473, loadavg=[0, 0, 0], uptime=6850.3204792, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:23:13.703Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:23:13 GMT-0300 (Hora estándar de Argentina), pid=7108, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134504448, heapTotal=110481408, heapUsed=72622768, external=19365473, loadavg=[0, 0, 0], uptime=6850.3264795, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:23:54.002Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:23:53 GMT-0300 (Hora estándar de Argentina), pid=4656, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134135808, heapTotal=111005696, heapUsed=73118280, external=19425771, loadavg=[0, 0, 0], uptime=6890.6227844, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:23:54.020Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:23:54 GMT-0300 (Hora estándar de Argentina), pid=4656, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134557696, heapTotal=111005696, heapUsed=73272424, external=19425771, loadavg=[0, 0, 0], uptime=6890.6437856, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:29:19.797Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:29:19 GMT-0300 (Hora estándar de Argentina), pid=6352, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134258688, heapTotal=110481408, heapUsed=72431464, external=19365635, loadavg=[0, 0, 0], uptime=7216.4204189, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:29:19.801Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:29:19 GMT-0300 (Hora estándar de Argentina), pid=6352, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134676480, heapTotal=110481408, heapUsed=72585568, external=19365635, loadavg=[0, 0, 0], uptime=7216.4244192, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:32:24.734Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:32:24 GMT-0300 (Hora estándar de Argentina), pid=5884, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=138678272, heapTotal=111529984, heapUsed=83065256, external=20963384, loadavg=[0, 0, 0], uptime=7401.3559966, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:32:24.738Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:32:24 GMT-0300 (Hora estándar de Argentina), pid=5884, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=139063296, heapTotal=111529984, heapUsed=83211680, external=20963384, loadavg=[0, 0, 0], uptime=7401.361997, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:34:14.688Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:34:14 GMT-0300 (Hora estándar de Argentina), pid=7036, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134291456, heapTotal=110481408, heapUsed=73814696, external=19426229, loadavg=[0, 0, 0], uptime=7511.3062854, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:34:14.695Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:34:14 GMT-0300 (Hora estándar de Argentina), pid=7036, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134692864, heapTotal=111005696, heapUsed=73968800, external=19426229, loadavg=[0, 0, 0], uptime=7511.316286, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:36:17.308Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:36:17 GMT-0300 (Hora estándar de Argentina), pid=1400, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134623232, heapTotal=111005696, heapUsed=72857152, external=19396248, loadavg=[0, 0, 0], uptime=7633.9292991, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:36:17.315Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:36:17 GMT-0300 (Hora estándar de Argentina), pid=1400, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135008256, heapTotal=111005696, heapUsed=73011248, external=19396248, loadavg=[0, 0, 0], uptime=7633.9382996, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:36:59.796Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:36:59 GMT-0300 (Hora estándar de Argentina), pid=6640, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134578176, heapTotal=110481408, heapUsed=72470768, external=19365436, loadavg=[0, 0, 0], uptime=7676.4187293, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:36:59.800Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:36:59 GMT-0300 (Hora estándar de Argentina), pid=6640, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134946816, heapTotal=110481408, heapUsed=72624624, external=19365436, loadavg=[0, 0, 0], uptime=7676.4237296, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:40:40.593Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:40:40 GMT-0300 (Hora estándar de Argentina), pid=6820, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135409664, heapTotal=110481408, heapUsed=73632336, external=19400127, loadavg=[0, 0, 0], uptime=7897.2153582, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:40:40.597Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:40:40 GMT-0300 (Hora estándar de Argentina), pid=6820, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135786496, heapTotal=110481408, heapUsed=73777512, external=19400127, loadavg=[0, 0, 0], uptime=7897.2203585, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:44:58.235Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:44:58 GMT-0300 (Hora estándar de Argentina), pid=6988, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134086656, heapTotal=109957120, heapUsed=73366984, external=19397136, loadavg=[0, 0, 0], uptime=8154.8560944, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:44:58.238Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:44:58 GMT-0300 (Hora estándar de Argentina), pid=6988, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134463488, heapTotal=109957120, heapUsed=73525432, external=19397136, loadavg=[0, 0, 0], uptime=8154.8630948, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:46:16.374Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:46:16 GMT-0300 (Hora estándar de Argentina), pid=6344, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=134680576, heapTotal=111005696, heapUsed=73325400, external=19426060, loadavg=[0, 0, 0], uptime=8232.9935636, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:46:16.382Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:46:16 GMT-0300 (Hora estándar de Argentina), pid=6344, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135069696, heapTotal=111005696, heapUsed=73479504, external=19426060, loadavg=[0, 0, 0], uptime=8233.0045642, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:51:27.769Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:51:27 GMT-0300 (Hora estándar de Argentina), pid=7160, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=136658944, heapTotal=111529984, heapUsed=74141016, external=19429971, loadavg=[0, 0, 0], uptime=8544.3923746, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:51:27.773Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:51:27 GMT-0300 (Hora estándar de Argentina), pid=7160, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=137076736, heapTotal=111529984, heapUsed=74286296, external=19429971, loadavg=[0, 0, 0], uptime=8544.3963748, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:52:41.777Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:52:41 GMT-0300 (Hora estándar de Argentina), pid=4672, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=132186112, heapTotal=108908544, heapUsed=78647344, external=19840747, loadavg=[0, 0, 0], uptime=8618.3976074, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:52:41.784Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:52:41 GMT-0300 (Hora estándar de Argentina), pid=4672, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=132562944, heapTotal=108908544, heapUsed=78801032, external=19840747, loadavg=[0, 0, 0], uptime=8618.4056079, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T14:54:10.471Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Thu May 11 2017 11:54:10 GMT-0300 (Hora estándar de Argentina), pid=1704, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=136089600, heapTotal=111005696, heapUsed=74838512, external=19430249, loadavg=[0, 0, 0], uptime=8707.0936806, trace=[column=11, file=util.js, function=Object.exports._errnoException, line=1033, method=exports._errnoException, native=false, column=20, file=util.js, function=exports._exceptionWithHostPort, line=1056, method=_exceptionWithHostPort, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1105, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object.exports._errnoException (util.js:1033:11),     at exports._exceptionWithHostPort (util.js:1056:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1105:14)]
2017-05-11T14:54:10.474Z - error: uncaughtException: socket hang up date=Thu May 11 2017 11:54:10 GMT-0300 (Hora estándar de Argentina), pid=1704, uid=null, gid=null, cwd=C:\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=136495104, heapTotal=111005696, heapUsed=74983568, external=19430249, loadavg=[0, 0, 0], uptime=8707.0986808, trace=[column=15, file=_http_client.js, function=createHangUpError, line=302, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=334, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=101, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=191, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=513, method=_handle.close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:302:15),     at Socket.socketCloseListener (_http_client.js:334:23),     at emitOne (events.js:101:20),     at Socket.emit (events.js:191:7),     at TCP._handle.close [as _onclose] (net.js:513:12)]
2017-05-11T15:03:02.703Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: post_data is not defined ReferenceError: post_data is not defined
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:163:47
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:04:44.853Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: "string" must be a string, Buffer, or ArrayBuffer TypeError: "string" must be a string, Buffer, or ArrayBuffer
    at Function.byteLength (buffer.js:363:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:174:36
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:09:47.154Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: First argument must be a string or Buffer TypeError: First argument must be a string or Buffer
    at ClientRequest.write (_http_outgoing.js:502:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:182:8
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:09:50.295Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: First argument must be a string or Buffer TypeError: First argument must be a string or Buffer
    at ClientRequest.write (_http_outgoing.js:502:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:182:8
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:09:55.382Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: First argument must be a string or Buffer TypeError: First argument must be a string or Buffer
    at ClientRequest.write (_http_outgoing.js:502:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:182:8
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:11:46.664Z - error: Error during processing request: http://localhost:3000/reporting/rpt details: http.request(...).on(...).write(...).end is not a function TypeError: http.request(...).on(...).write(...).end is not a function
    at C:\tp_profesional\FlourApp-TrabajoProfesional\server_app.js:182:25
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at expressInit (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\init.js:40:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at query (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\middleware\query.js:44:5)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:174:10)
    at mounted_app (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\application.js:230:10)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:635:15
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:174:3)
    at router (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\serve-static\index.js:121:7)
    at emitOne (events.js:96:13)
    at SendStream.emit (events.js:191:7)
    at SendStream.error (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:282:17)
    at SendStream.onStatError (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:433:12)
    at next (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:743:16)
    at onstat (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\send\index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:114:15)
2017-05-11T15:17:16.100Z - error: Error during processing request: http://localhost:3000/reporting/api/report/Factura details: Could not parse report template, aren't you missing content type? undefined
2017-05-11T15:26:14.450Z - error: Error when processing render request Error during rendering report: Engine must be specified Error: Engine must be specified
    at invokeRender (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\render.js:66:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\render.js:146:12
    at tryCatcher (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:510:31)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at Promise._resolveCallback (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:431:57)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:522:17)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at Promise._resolveCallback (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:431:57)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:522:17)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at PromiseArray._resolve (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise_array.js:125:19)
    at PromiseArray._promiseFulfilled (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise_array.js:143:14)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:572:26)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Async._drainQueue (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:666:20)
    at tryOnImmediate (timers.js:639:5)
    at processImmediate [as _immediateCallback] (timers.js:611:5) requestId=1
2017-05-11T15:26:14.475Z - error: Error during processing request: http://localhost:3001/api/report details: Error during rendering report: Engine must be specified Error: Engine must be specified
    at invokeRender (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\render.js:66:11)
    at C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\render.js:146:12
    at tryCatcher (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:510:31)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at Promise._resolveCallback (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:431:57)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:522:17)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at Promise._resolveCallback (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:431:57)
    at Promise._settlePromiseFromHandler (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:522:17)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:567:18)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Promise._fulfill (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:636:18)
    at PromiseArray._resolve (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise_array.js:125:19)
    at PromiseArray._promiseFulfilled (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise_array.js:143:14)
    at Promise._settlePromise (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:572:26)
    at Promise._settlePromise0 (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:612:10)
    at Promise._settlePromises (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\promise.js:691:18)
    at Async._drainQueue (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (C:\tp_profesional\FlourApp-TrabajoProfesional\node_modules\listener-collection\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:666:20)
    at tryOnImmediate (timers.js:639:5)
    at processImmediate [as _immediateCallback] (timers.js:611:5)
2017-05-14T21:43:11.838Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 18:43:11 GMT-0300 (Hora estándar de Argentina), pid=2576, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135872512, heapTotal=109957120, heapUsed=72946816, external=20049235, loadavg=[0, 0, 0], uptime=36705.1357415, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=248, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:248:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T21:44:48.302Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 18:44:48 GMT-0300 (Hora estándar de Argentina), pid=6140, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133230592, heapTotal=109432832, heapUsed=73021968, external=19925035, loadavg=[0, 0, 0], uptime=36801.5996087, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T21:45:51.050Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 18:45:51 GMT-0300 (Hora estándar de Argentina), pid=5576, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=86806528, heapTotal=67039232, heapUsed=44534144, external=17882562, loadavg=[0, 0, 0], uptime=36864.3476556, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T21:46:39.009Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 18:46:38 GMT-0300 (Hora estándar de Argentina), pid=1384, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135139328, heapTotal=109432832, heapUsed=74717768, external=19958366, loadavg=[0, 0, 0], uptime=36912.3056634, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:24:10.996Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 19:24:10 GMT-0300 (Hora estándar de Argentina), pid=1608, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=99160064, heapTotal=75501568, heapUsed=63060728, external=17956118, loadavg=[0, 0, 0], uptime=39164.2919915, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:31:50.094Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 19:31:50 GMT-0300 (Hora estándar de Argentina), pid=5868, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=132911104, heapTotal=109432832, heapUsed=73099624, external=20045734, loadavg=[0, 0, 0], uptime=39623.3906244, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:33:27.543Z - error: uncaughtException: datosPropios is not defined date=Sun May 14 2017 19:33:27 GMT-0300 (Hora estándar de Argentina), pid=4860, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=135094272, heapTotal=109432832, heapUsed=74328176, external=20049235, loadavg=[0, 0, 0], uptime=39720.8398431, trace=[column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=264, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datosPropios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:264:21,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:34:06.759Z - error: uncaughtException: documentosmercantiles is not defined date=Sun May 14 2017 19:34:06 GMT-0300 (Hora estándar de Argentina), pid=4824, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133558272, heapTotal=109432832, heapUsed=72631576, external=19880375, loadavg=[0, 0, 0], uptime=39760.05664, trace=[column=27, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=299, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: documentosmercantiles is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:299:27,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:38:15.404Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 19:38:15 GMT-0300 (Hora estándar de Argentina), pid=5488, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=132812800, heapTotal=107859968, heapUsed=77883040, external=21406391, loadavg=[0, 0, 0], uptime=40008.7011712, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-14T22:41:23.335Z - error: uncaughtException: facturacionDatosPropiosModel.findAll is not a function date=Sun May 14 2017 19:41:23 GMT-0300 (Hora estándar de Argentina), pid=5552, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133197824, heapTotal=108908544, heapUsed=72212096, external=20045324, loadavg=[0, 0, 0], uptime=40196.6318353, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=252, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: facturacionDatosPropiosModel.findAll is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:252:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T00:03:12.293Z - error: Error when processing render request Error during rendering report: Cannot read property 'razonSocial' of undefined TypeError: Cannot read property 'razonSocial' of undefined
    at Function.eval [as fn] (eval at buildCode (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsrender\jsrender-node.js:2156:10), <anonymous>:9:24)
    at Function.renderContent [as render] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsrender\jsrender-node.js:1384:20)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-jsrender\lib\jsrenderEngine.js:22:19
    at evalmachine.<anonymous>:1:41
    at ContextifyScript.Script.runInContext (vm.js:32:29)
    at ContextifyScript.Script.runInNewContext (vm.js:38:15)
    at Object.runInNewContext (vm.js:91:38)
    at module.exports (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\engineScript.js:185:8)
    at process.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\script-manager\lib\worker-processes.js:48:36)
    at emitTwo (events.js:106:13) requestId=2
2017-05-15T00:03:12.297Z - error: Error during processing request: http://localhost:3001/api/report details: Error during rendering report: Cannot read property 'razonSocial' of undefined TypeError: Cannot read property 'razonSocial' of undefined
    at Function.eval [as fn] (eval at buildCode (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsrender\jsrender-node.js:2156:10), <anonymous>:9:24)
    at Function.renderContent [as render] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsrender\jsrender-node.js:1384:20)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-jsrender\lib\jsrenderEngine.js:22:19
    at evalmachine.<anonymous>:1:41
    at ContextifyScript.Script.runInContext (vm.js:32:29)
    at ContextifyScript.Script.runInNewContext (vm.js:38:15)
    at Object.runInNewContext (vm.js:91:38)
    at module.exports (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\render\engineScript.js:185:8)
    at process.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\script-manager\lib\worker-processes.js:48:36)
    at emitTwo (events.js:106:13)
2017-05-15T00:03:33.661Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 21:03:33 GMT-0300 (Hora estándar de Argentina), pid=5560, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=136069120, heapTotal=109957120, heapUsed=73659864, external=19884903, loadavg=[0, 0, 0], uptime=45126.9580072, trace=[column=30, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=277, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:277:30,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T00:04:42.535Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 21:04:42 GMT-0300 (Hora estándar de Argentina), pid=3500, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133562368, heapTotal=109957120, heapUsed=72724456, external=19939702, loadavg=[0, 0, 0], uptime=45195.8320306, trace=[column=30, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=277, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:277:30,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:27:24.179Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:27:24 GMT-0300 (Hora estándar de Argentina), pid=4764, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133742592, heapTotal=108384256, heapUsed=75913424, external=21204659, loadavg=[0, 0, 0], uptime=53757.4746087, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=277, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:277:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:28:02.291Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:28:02 GMT-0300 (Hora estándar de Argentina), pid=1936, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=130719744, heapTotal=107335680, heapUsed=77884888, external=21416767, loadavg=[0, 0, 0], uptime=53795.58789, trace=[column=36, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=277, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:277:36,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:29:05.252Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:29:05 GMT-0300 (Hora estándar de Argentina), pid=5192, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=132628480, heapTotal=108384256, heapUsed=79218096, external=21257439, loadavg=[0, 0, 0], uptime=53858.5488275, trace=[column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=266, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:266:21,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:29:50.594Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:29:50 GMT-0300 (Hora estándar de Argentina), pid=5432, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=130740224, heapTotal=107335680, heapUsed=74520160, external=21185907, loadavg=[0, 0, 0], uptime=53903.8925775, trace=[column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=266, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:266:21,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:30:29.184Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:30:29 GMT-0300 (Hora estándar de Argentina), pid=1608, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=128532480, heapTotal=106811392, heapUsed=71903112, external=20608940, loadavg=[0, 0, 0], uptime=53942.4814447, trace=[column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=266, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:266:21,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:30:52.966Z - error: uncaughtException: datos_propios is not defined date=Sun May 14 2017 23:30:52 GMT-0300 (Hora estándar de Argentina), pid=3708, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=133455872, heapTotal=109432832, heapUsed=72511192, external=19850171, loadavg=[0, 0, 0], uptime=53966.2646478, trace=[column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=268, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: datos_propios is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:268:21,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T02:44:19.639Z - error: Error during processing request: http://localhost:3001/api/report details: Not initialized, you need to call jsreport.init().then before rendering Error: Not initialized, you need to call jsreport.init().then before rendering
    at Reporter.render (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\reporter.js:168:11)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:90:14
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:49:5
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-studio\lib\studio.js:69:7
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at cors (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:185:7)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:221:17
    at originCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:211:15)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:216:13
    at optionsCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:196:9)
    at corsMiddleware (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:201:7)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\body-parser\lib\read.js:129:5
    at invokeCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:262:16)
    at done (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:251:7)
    at IncomingMessage.onEnd (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:307:7)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
2017-05-15T03:24:48.645Z - error: uncaughtException: connection timeout date=Mon May 15 2017 00:24:48 GMT-0300 (Hora estándar de Argentina), pid=6444, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=125652992, heapTotal=103813120, heapUsed=71366968, external=18789472, loadavg=[0, 0, 0], uptime=57201.9394525, trace=[column=17, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js, function=, line=169, method=null, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Db.emit, line=194, method=emit, native=false, column=14, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb\lib\db.js, function=Server.listener, line=1798, method=listener, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Server.emit, line=191, method=emit, native=false, column=14, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb\lib\server.js, function=, line=274, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Server.emit, line=191, method=emit, native=false, column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\topologies\server.js, function=, line=335, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=191, method=emit, native=false, column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\connection\pool.js, function=, line=270, method=null, native=false, column=19, file=events.js, function=Object.onceWrapper, line=293, method=onceWrapper, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=194, method=emit, native=false, column=10, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\connection\connection.js, function=, line=185, method=null, native=false, column=19, file=events.js, function=Object.onceWrapper, line=293, method=onceWrapper, native=false, column=13, file=events.js, function=emitNone, line=86, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=Socket._onTimeout, line=354, method=_onTimeout, native=false, column=14, file=timers.js, function=ontimeout, line=380, method=null, native=false, column=5, file=timers.js, function=tryOnTimeout, line=244, method=null, native=false, column=5, file=timers.js, function=Timer.listOnTimeout, line=214, method=listOnTimeout, native=false], stack=[Error: connection timeout,     at Db.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\drivers\node-mongodb-native\connection.js:169:17),     at emitTwo (events.js:106:13),     at Db.emit (events.js:194:7),     at Server.listener (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb\lib\db.js:1798:14),     at emitOne (events.js:96:13),     at Server.emit (events.js:191:7),     at Server.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb\lib\server.js:274:14),     at emitOne (events.js:96:13),     at Server.emit (events.js:191:7),     at Pool.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\topologies\server.js:335:12),     at emitOne (events.js:96:13),     at Pool.emit (events.js:191:7),     at Connection.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\connection\pool.js:270:12),     at Object.onceWrapper (events.js:293:19),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:194:7),     at Socket.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongodb-core\lib\connection\connection.js:185:10),     at Object.onceWrapper (events.js:293:19),     at emitNone (events.js:86:13),     at Socket.emit (events.js:188:7),     at Socket._onTimeout (net.js:354:8),     at ontimeout (timers.js:380:14),     at tryOnTimeout (timers.js:244:5),     at Timer.listOnTimeout (timers.js:214:5)]
2017-05-15T03:39:52.940Z - error: uncaughtException: inicioActividades is not defined date=Mon May 15 2017 00:39:52 GMT-0300 (Hora estándar de Argentina), pid=5192, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=88915968, heapTotal=64987136, heapUsed=54274776, external=17965119, loadavg=[0, 0, 0], uptime=58106.2265619, trace=[column=29, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=267, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: inicioActividades is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:267:29,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T03:44:27.954Z - error: uncaughtException: datosPropios[0].inicioActividades.split is not a function date=Mon May 15 2017 00:44:27 GMT-0300 (Hora estándar de Argentina), pid=4088, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=88231936, heapTotal=64462848, heapUsed=54334240, external=17964914, loadavg=[0, 0, 0], uptime=58381.2509759, trace=[column=61, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=267, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: datosPropios[0].inicioActividades.split is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:267:61,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T03:56:32.422Z - error: Error during processing request: http://localhost:3001/api/report details: Not initialized, you need to call jsreport.init().then before rendering Error: Not initialized, you need to call jsreport.init().then before rendering
    at Reporter.render (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\reporter.js:168:11)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:90:14
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:49:5
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-studio\lib\studio.js:69:7
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at cors (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:185:7)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:221:17
    at originCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:211:15)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:216:13
    at optionsCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:196:9)
    at corsMiddleware (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:201:7)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\body-parser\lib\read.js:129:5
    at invokeCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:262:16)
    at done (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:251:7)
    at IncomingMessage.onEnd (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:307:7)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
2017-05-15T03:57:48.298Z - error: uncaughtException: datosPropios[0].inicioActividades.toLocalDateString is not a function date=Mon May 15 2017 00:57:48 GMT-0300 (Hora estándar de Argentina), pid=596, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=127356928, heapTotal=105910272, heapUsed=64394152, external=19663309, loadavg=[0, 0, 0], uptime=59181.5937494, trace=[column=61, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=267, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: datosPropios[0].inicioActividades.toLocalDateString is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:267:61,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T03:58:39.459Z - error: Error during processing request: http://localhost:3001/api/report details: Not initialized, you need to call jsreport.init().then before rendering Error: Not initialized, you need to call jsreport.init().then before rendering
    at Reporter.render (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\reporter.js:168:11)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:90:14
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:49:5
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-studio\lib\studio.js:69:7
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at cors (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:185:7)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:221:17
    at originCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:211:15)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:216:13
    at optionsCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:196:9)
    at corsMiddleware (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:201:7)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\body-parser\lib\read.js:129:5
    at invokeCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:262:16)
    at done (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:251:7)
    at IncomingMessage.onEnd (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:307:7)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
2017-05-15T04:00:39.195Z - error: uncaughtException: dateSplit.split is not a function date=Mon May 15 2017 01:00:39 GMT-0300 (Hora estándar de Argentina), pid=6484, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=130293760, heapTotal=106287104, heapUsed=74957976, external=19675602, loadavg=[0, 0, 0], uptime=59352.4931634, trace=[column=35, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=271, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: dateSplit.split is not a function,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:271:35,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:26:56.418Z - error: uncaughtException: documento is not defined date=Mon May 15 2017 01:26:56 GMT-0300 (Hora estándar de Argentina), pid=6684, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=87330816, heapTotal=62889984, heapUsed=53856648, external=17956312, loadavg=[0, 0, 0], uptime=60929.7138665, trace=[column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=264, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: documento is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:264:12,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:29:28.298Z - error: uncaughtException: documento is not defined date=Mon May 15 2017 01:29:28 GMT-0300 (Hora estándar de Argentina), pid=6708, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=130367488, heapTotal=106434560, heapUsed=64204808, external=19783603, loadavg=[0, 0, 0], uptime=61081.5937494, trace=[column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=262, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: documento is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:262:12,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:31:12.535Z - error: uncaughtException: documdocumentomercantilento is not defined date=Mon May 15 2017 01:31:12 GMT-0300 (Hora estándar de Argentina), pid=6808, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=131047424, heapTotal=107335680, heapUsed=76611632, external=19751813, loadavg=[0, 0, 0], uptime=61185.8320306, trace=[column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=266, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: documdocumentomercantilento is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:266:12,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:32:34.556Z - error: uncaughtException: documdocumentomercantilento is not defined date=Mon May 15 2017 01:32:34 GMT-0300 (Hora estándar de Argentina), pid=5100, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=130244608, heapTotal=106958848, heapUsed=64273392, external=19783808, loadavg=[0, 0, 0], uptime=61267.8525384, trace=[column=12, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=267, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[ReferenceError: documdocumentomercantilento is not defined,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:267:12,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:44:13.157Z - error: uncaughtException: Cannot read property 'toLocaleDateString' of undefined date=Mon May 15 2017 01:44:13 GMT-0300 (Hora estándar de Argentina), pid=6316, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=131313664, heapTotal=106958848, heapUsed=64290232, external=19889365, loadavg=[0, 0, 0], uptime=61966.4531244, trace=[column=25, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=toReadableDate, line=242, method=null, native=false, column=28, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=298, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: Cannot read property 'toLocaleDateString' of undefined,     at toReadableDate (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:242:25),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:298:28,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:45:22.572Z - error: uncaughtException: Cannot read property 'toLocaleDateString' of undefined date=Mon May 15 2017 01:45:22 GMT-0300 (Hora estándar de Argentina), pid=1688, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=87781376, heapTotal=67039232, heapUsed=40377352, external=17883491, loadavg=[0, 0, 0], uptime=62035.86914, trace=[column=25, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=toReadableDate, line=242, method=null, native=false, column=28, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=301, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: Cannot read property 'toLocaleDateString' of undefined,     at toReadableDate (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:242:25),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:301:28,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:46:26.743Z - error: uncaughtException: Cannot read property 'toLocaleDateString' of undefined date=Mon May 15 2017 01:46:26 GMT-0300 (Hora estándar de Argentina), pid=5432, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=129363968, heapTotal=106434560, heapUsed=64081984, external=19722139, loadavg=[0, 0, 0], uptime=62100.0400384, trace=[column=25, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=toReadableDate, line=242, method=null, native=false, column=28, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=302, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: Cannot read property 'toLocaleDateString' of undefined,     at toReadableDate (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:242:25),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:302:28,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:49:36.584Z - error: uncaughtException: Cannot read property 'toLocaleDateString' of undefined date=Mon May 15 2017 01:49:36 GMT-0300 (Hora estándar de Argentina), pid=6416, uid=null, gid=null, cwd=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional, execPath=C:\Program Files\nodejs\node.exe, version=v7.7.2, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\server_app.js], rss=89767936, heapTotal=64987136, heapUsed=54751224, external=17965649, loadavg=[0, 0, 0], uptime=62289.8798822, trace=[column=25, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=toReadableDate, line=242, method=null, native=false, column=28, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js, function=null, line=304, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js, function=model.Query.<anonymous>, line=3433, method=Query.<anonymous>, native=false, column=21, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=273, method=null, native=false, column=16, file=C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js, function=null, line=127, method=null, native=false, column=7, file=internal/process/next_tick.js, function=_combinedTickCallback, line=73, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[TypeError: Cannot read property 'toLocaleDateString' of undefined,     at toReadableDate (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:242:25),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\controllers\documentoMercantilController.js:304:28,     at model.Query.<anonymous> (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\mongoose\lib\model.js:3433:16),     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:273:21,     at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\kareem\index.js:127:16,     at _combinedTickCallback (internal/process/next_tick.js:73:7),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2017-05-15T04:53:22.762Z - error: Error during processing request: http://localhost:3001/api/report details: Not initialized, you need to call jsreport.init().then before rendering Error: Not initialized, you need to call jsreport.init().then before rendering
    at Reporter.render (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-core\lib\reporter.js:168:11)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:90:14
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-express\lib\routes.js:49:5
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\jsreport-studio\lib\studio.js:69:7
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at cors (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:185:7)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:221:17
    at originCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:211:15)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:216:13
    at optionsCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:196:9)
    at corsMiddleware (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\cors\lib\index.js:201:7)
    at Layer.handle [as handle_request] (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\express\lib\router\index.js:275:10)
    at C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\body-parser\lib\read.js:129:5
    at invokeCallback (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:262:16)
    at done (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:251:7)
    at IncomingMessage.onEnd (C:\Users\Euclides\Desktop\tp_profesional\FlourApp-TrabajoProfesional\node_modules\raw-body\index.js:307:7)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
